server/gokbg3/grails-app/domain/org/gokb/cred/Office.groovy

```diff
@@ -51,8 +56,13 @@ class Office extends KBComponent {
 	region (nullable:true, blank:true)
 	state (nullable:true, blank:true)
 	country (nullable:true, blank:true)
+  function (nullable: true, blank: false)
   }
 
+  private static refdataDefaults = [
+      "Office.Function"		: "Technical Support"
```

Comment:
My previous statement was wrong. Since the property is defined locally and not in a parent class, just using `function` as the key is correct here.

Reply:
did it.